aiccu: fix undefined reference to clock_* functions

Since glibc 2.17 the clock_* suite functions are available directly in 
the main C library. For previous versions we still need -lrt to link.

Signed-off-by: Vicente Olivert Riera <vincent.riera@imgtec.com>

--- aiccu/unix-console/Makefile.orig	2014-01-14 10:17:20.204602513 +0000
+++ aiccu/unix-console/Makefile	2014-01-14 10:17:31.435777359 +0000
@@ -42,7 +42,7 @@ CFLAGS	+= -D AICCU_CONSOLE
 # Currently defaultly builds only on Linux, but other platforms might easily also support it
 ifeq ($(shell uname | grep -c "Linux"),1)
 CFLAGS	+= -D AICCU_GNUTLS
-LDFLAGS += -lgnutls
+LDFLAGS += -lgnutls -lrt
 endif
 
 # Linux
